lib/spritesheet-templates.js

```diff
@@ -315,6 +315,8 @@ templater.addTemplate('json', require(templatesDir + '/json.template.js'));
 templater.addTemplate('jsonArray', require(templatesDir + '/json_array.template.js'));
 templater.addTemplate('css', require(templatesDir + '/css.template.js'));
 
+templater.addTemplate('jsonTexture', require(templatesDir + '/json_texture.template.js'));
```

Comment:
`jsonArray` is camelCased due to legacy reasons. We typically go with `snake_case` for template names now (e.g. `scss_maps`). Can we update this name to `json_texture`?